test/helpers/assertions.js

```diff
@@ -140,3 +143,38 @@ function assertContractSaved(expectedAddress, expectedChain, expectedStatus) {
     chai.expect(isExist, "Contract is not saved").to.be.true;
   }
 }
+
+/**
+ * assertContractNotSaved checks that the verification result was not stored on the file system.
+ */
+const assertContractNotSaved = (expectedAddress, expectedChain) => {
+  // Check contract verification result was NOT saved to the disk
+  const fullMatchPath = path.join(
+    config.repository.path,
+    "contracts",
+    "full_match",
+    expectedChain,
+    getAddress(expectedAddress),
+    "metadata.json"
+  )
+  const partialMatchPath = path.join(
+    config.repository.path,
+    "contracts",
+    "partial_match",
+    expectedChain,
+    getAddress(expectedAddress),
+    "metadata.json"
+  )
+  const isExist = fs.existsSync(fullMatchPath) || fs.existsSync(partialMatchPath);
+  chai.expect(isExist, "Contract should not have been saved").to.be.false;
+}
+
+module.exports = {
+  assertValidationError,
+  assertVerification,
+  assertVerificationSession,
+  assertLookup,
+  assertLookupAll,
+  assertContractSaved,
+  assertContractNotSaved
+}
```

Comment:
Just wondering, is there a specific reason why `export` everything at the end? It might be more declarative to use `exports.[...]` syntax.

Reply:
I did that initially and with the other functions of the module using assertContractSaved, I am getting a Chai error complaining that assertContractSaved is not defined....

Reply:
Just FYI, if you use `exports.assertContractSaved` (instead of `assertContractSaved` alone) should work. For this to work, you should remove `module.exports =` because I believe it overwrites the `exports.` assignment,

In any case, probably only a matter of style, so whatever you prefer.

Reply:
I meant: this is exactly what I did first (i.e. keeping the declaration style present it the file) but that gives a Chai error when the function is called inside the module. So I suggest to leave it like that for now.